offline-virtual-machine.md

```diff
@@ -39,36 +104,35 @@ kubectl describe offlinevirtualmachine myvm
 # Look at the now created VirtualMachine status and associated events:
 kubectl describe virtualmachine myvm
 
-# Stop an OfflineVirtualMachine:
-kubectl patch offlinevirtualmachine myvm -p \
+# Stop the VirtualMachine created by this OfflineVirtualMachine:
+kubectl patch offlinevirtualmachine myvm --type merge -p \
     '{"spec":{"running":"false"}}'
 
-# Implicit cascade delete (first deletes the vm and then the ovm)
+# Implicit cascade delete (first deletes the VirtualMachine and then the OfflineVirtualMachine)
 kubectl delete offlinevirtualmachine myvm
 
-# Explicit cascade delete (first deletes the vm and then the ovm)
+# Explicit cascade delete (first deletes the VirtualMAchine and then the OfflineVirtualMachine)
 kubectl delete offlinevirtualmachine myvm --cascade=true
 
-# Orphan delete (The running vm is only detached, not deleted)
-# Recreating the ovm would lead to the adoption of the vm
+# Orphan delete (The running VirtualMachine is only detached, not deleted)
+# Recreating the OfflineVirtualMachine would lead to the adoption of the VirtualMachine
 kubectl delete offlinevirtualmachine myvm --cascade=false
 ```
 
-### REST API
+## What it does
```

Comment:
`How it works - The realtionship between VirtualMachine and OfflineVirtulMachine`